src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -307,4 +314,24 @@ private void addJurCodeToExistingCollection(CaseData caseData, VettingJurisdicti
         caseData.getJurCodesCollection().add(codesTypeItem);
     }
 
+    public DynamicFixedListType getHearingVenuesList(CaseData caseData) {
+        var dynamicListingVenues = new DynamicFixedListType();
+
+        dynamicListingVenues.setListItems(new ArrayList<>(
```

Comment:
Can we not use List.of()

Reply:
Even better, setListItems requires a list and that's what's returned by valueOfOfficeName. No need for a new list.